rllib/algorithms/dqn/dqn.py

```diff
@@ -221,7 +222,6 @@ def training(
                 "type": "MultiAgentReplayBuffer",
                 "learning_starts": 1000,
                 "capacity": 50000,
-                "replay_batch_size": 32,
```

Comment:
Question: So this setting is completely removed now and we always require the `sample()` size (normally via `train_batch_size`)?
If yes, can we make sure that setting `replay_batch_size` produces an error?